cmd/mattermost/commands/config.go

```diff
@@ -217,6 +229,45 @@ func configSetCmdF(command *cobra.Command, args []string) error {
 	return nil
 }
 
+func configMigrateCmdF(command *cobra.Command, args []string) error {
+	// Parse source config; defaults to global --config unless overwritten by --from
+	from, err := command.Flags().GetString("from")
+	if err != nil {
+		return errors.Wrap(err, "failed reading source config parameter")
+	}
+	if from == "" {
+		from = viper.GetString("config")
+	}
+
+	// Parse destination config store
+	to, err := command.Flags().GetString("to")
+	if err != nil {
+		return errors.Wrap(err, "failed reading destination parameter (--to)")
+	}
+	if to == "" {
```

Comment:
nit: maybe we want to mark it as "required" flag in cobra (https://github.com/spf13/cobra#required-flags